src/pytorch_lightning/utilities/seed.py

```diff
@@ -153,3 +151,14 @@ def isolate_rng() -> Generator[None, None, None]:
     states = _collect_rng_states()
     yield
     _set_rng_states(states)
+
+
+def _log_info(message: str) -> None:
+    """Check global rank if the rank is greater than zero, add a prefix."""
+    rank = _get_rank()
+
+    if rank > 0:
+        log.info(f"[rank: {rank}] {message}")
+        return
```

Comment:
Why are we excluding rank 0?

Reply:
Hi, thanks for watching! I read your comment to [issue](https://github.com/Lightning-AI/lightning/issues/13290#issuecomment-1159567609) that in some cases, _get_rank() might the prefix will falsely display rank 0 for all messages. I thought that maybe in such cases we can not change the logging behavior.

Do you think it's better to always specify rank in logging? I can do so.

Reply:
Yes, on some custom clusters `_get_rank()` will return 0 at the time of calling `seed_everything()`, because the true rank information is not yet available. You can't avoid this, even when you do a check for `rank > 0`, it won't be true and so can't address this problem. It's not a huge issue, and we can't really do anything about it. 

> Do you think it's better to always specify rank in logging? I can do so

yes

Reply:
Got it! Thanks for the explanation, I will make changes